includes/internal/emails/class-sensei-email-sender.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * File containing the Sensei_Email_Sender class.
+ *
+ * @package sensei
+ */
+
+namespace Sensei\Internal\Emails;
+
+// phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis -- Prevent "Unused global variable $sensei_email_data"
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+
+require plugin_dir_path( __DIR__ ) . '../../vendor/autoload.php';
```

Comment:
Hey @Imran92 I'm trying to understand what autoload file are we loading here? Is it the composer autoload of the WP instance?

Reply:
Hey 👋 Yes, otherwise it didn't load the vendor libs